component-classes/index.js

```diff
@@ -207,14 +207,14 @@ export const expandable = {
   expandableBleed: box.bleed,
   chevron: 'inline-block align-middle i-text-$color-expandable-icon',
   chevronNonBox: 'relative left-8',
```

Comment:
Now that we don't apply `absolute` position on chevronBox, `chevronNonBox` can be changed to `ml-8`. This also fixes the following situation in Elements:
Before:
<img width="163" alt="Screenshot 2023-11-13 at 14 48 34" src="https://github.com/warp-ds/css/assets/41303231/60a4cfe3-f880-4585-8166-39053b33b5e9">

After:
<img width="162" alt="Screenshot 2023-11-13 at 14 50 01" src="https://github.com/warp-ds/css/assets/41303231/ae3b7f32-66f2-40f6-9d5f-3b0f0a479a1f">

Reply:
[f690a2d](https://github.com/warp-ds/css/pull/89/commits/f690a2d864699570607a2f69abb40839aa1976c8)